platform/ui/src/components/MeasurementsContextMenu/MeasurementsContextMenu.jsx

```diff
@@ -0,0 +1,56 @@
+import { ContextMenu } from '@ohif/ui';
```

Comment:
Can we rename this component `ContextMenuMeasurements`
It's silly, I know.

Reply:
Sure! -- renamed